impy/kinematics.py

```diff
@@ -109,7 +199,20 @@ def __repr__(self):
         return ostr
 
 
-class EventKinematics(object):
+def get_particle_input_type(arg):
+    if isinstance(arg, int):
```

Comment:
I believe this is also a private function so should start with `_`

Reply:
Yes. Changed.